### ISSUE TITLE
Issue when trying to run container

### ISSUE THREAD

**Issue by Viky397** [jonazpiazu/docker_images#2]
Hello,

When I run `sudo docker run -it jonazpiazu/elastic_fusion`, I get the following error: 
`ElasticFusion: error while loading shared libraries: libefusion.so: cannot open shared object file: No such file or directory
`
Would you happen to know how to get around this? I was able to clone and build the repo. Thank you!

**Comment by jonazpiazu** [comment]
First of all, note that the repository is just intended for testing and proof of concept, so please do not expect it to be bullet-proof.

The library is installed at `/usr/local/lib/libefusion.so` , and that folder is included at:

```
cat /etc/ld.so.conf.d/local.conf 
/usr/local/lib
```

So it looks like after installing that library, `ldconfig` should be run to rebuild the cache.

**Comment by Viky397** [comment]
Thank you for that!

When I actually try to run Elastic fusion with their sample dataset: `./ElasticFusion -l /src/ElasticFusion/dyson_lab.klg
`

I run into the following issue: `terminate called after throwing an instance of 'std::runtime_error'
  what():  Pangolin X11: Unable to retrieve framebuffer options
Aborted (core dumped)
`

Have you run into this issue before? I've tried some of the recommended fixes I've found online, to no avail. Thank you!
